app/src/main/java/fr/free/nrw/commons/media/MediaDetailPagerFragment.java

```diff
@@ -414,5 +424,30 @@ public int getCount() {
             }
             return provider.getTotalMediaCount();
         }
+
+        /**
+         * getter for mCurrentFragment
```

Comment:
-> `Get the currently displayed fragment.`
Javadoc is written once (usually) and read by many developers (and shown by IDEs), so it is worth the effort :-)

Reply:
done, Thanks